src/main/java/tests/uiTests/CheckFormatForSelectedStatesTest.java

```diff
@@ -0,0 +1,21 @@
+package tests.uiTests;
+
+import domain.pages.DirectoryPage;
+import domain.pages.HomePage;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.support.PageFactory;
+
+public class CheckFormatForSelectedStatesTest extends UITestBaseClass{
+
+    @Test
+    public void checkFormat(){
+        HomePage homePage = new PageFactory().initElements(driver, HomePage.class);
+        DirectoryPage directoryPage = new PageFactory().initElements(driver, DirectoryPage.class);
+        homePage.validateHomePage();
```

Comment:
first you should land on the home page.with driver, get()

Reply:
Already done with UITestBaseClass